src/worker/event/lambdaServer.go

```diff
@@ -89,14 +166,18 @@ func NewLambdaServer() (*LambdaServer, error) {
 	server := &LambdaServer{
 		lambdaMgr: lambdaMgr,
 	}
-
-	log.Printf("Setups Handlers")
-	port := fmt.Sprintf(":%s", common.Conf.Worker_port)
-	http.HandleFunc(RUN_PATH, server.RunLambda)
-	http.HandleFunc(DEBUG_PATH, server.Debug)
-
-	log.Printf("Execute handler by POSTing to localhost%s%s%s\n", port, RUN_PATH, "<lambda>")
-	log.Printf("Get status by sending request to localhost%s%s\n", port, STATUS_PATH)
+	switch common.Conf.Trigger {
```

Comment:
Let's not be required to do one type or the other at the granularity of a worker.  I'm imagining a function config can indicate it should be triggered by Kafka, HTTP, or both.